python/ray/train/torch/config.py

```diff
@@ -14,6 +14,11 @@
 from ray.train.constants import DEFAULT_NCCL_SOCKET_IFNAME
 from ray.util import PublicAPI
 
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
+if HPU_PACKAGE_AVAILABLE:
+    import habana_frameworks.torch.core as htcore
+    import habana_frameworks.torch.distributed.hccl as hpu_dist
```

Comment:
Where did you use `htcore` and `hpu_dist` after import?  Any environment variable need to be set for hccl backend?

Reply:
htcore is imported to initialize the habana PyTorch backend; hccl is imported to initialize the habana distributed backend.  Environment variable is not required. This is documented [here](https://docs.habana.ai/en/latest/PyTorch/PyTorch_Model_Porting/Porting_PyTorch_Models_to_Gaudi.html?highlight=htcore#importing-habana-frameworks-torch-core)